pkg/install/install.go

```diff
@@ -234,7 +234,7 @@ func (i *Installer) runSteps(ctx context.Context, steps []interface{}) error {
 					err = step(ctx)
 					if azureerrors.HasAuthorizationFailedError(err) ||
 						azureerrors.HasLinkedAuthorizationFailedError(err) {
-						i.log.Print(err)
+						i.log.Printf("found link authorization errors, will retry (%s)", err)
```

Comment:
I really think this is redundant. We already know that PRIORITY=6 is retryable error. Why we saying it again?